### ISSUE TITLE
ClientNotFoundError: 404 Client Error for challenge

### ISSUE THREAD

**Issue by quintendewilde** [adw0rd/instagrapi#117]
Hi I'm running the medias.py test with my account and I can't seem to get it work? 

Running this on terminal on mac. username & passwords are correct. I don't know where to get the cookie...?
I'm getting this error: 

```
2021-03-14 15:37:07,173 INFO public_request: _kyanrehorst_ [404] POST https://i.instagram.com/api/challenge/7182357309/K2scazkHeQ/ (169.3.0.30.135, Xiaomi capricorn)
2021-03-14 15:37:07,173 WARNING instagrapi: Status 404: Endpoint /challenge/7182357309/K2scazkHeQ/ does not exists
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/instagrapi/mixins/private.py", line 173, in _send_private_request
    response.raise_for_status()
  File "/usr/local/lib/python3.9/site-packages/requests/models.py", line 943, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 400 Client Error: Bad Request for url: https://i.instagram.com/api/v1/accounts/login/

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/instagrapi/mixins/private.py", line 293, in private_request
    self._send_private_request(endpoint, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/instagrapi/mixins/private.py", line 203, in _send_private_request
    raise ChallengeRequired(**last_json)
instagrapi.exceptions.ChallengeRequired: challenge_required

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/instagrapi/mixins/private.py", line 173, in _send_private_request
    response.raise_for_status()
  File "/usr/local/lib/python3.9/site-packages/requests/models.py", line 943, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://i.instagram.com/api/challenge/7182357309/K2scazkHeQ/

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/quinten/instagrapi/examples/medias.py", line 112, in <module>
    cl.login(IG_USERNAME, IG_PASSWORD)
  File "/usr/local/lib/python3.9/site-packages/instagrapi/mixins/auth.py", line 341, in login
    if self.private_request("accounts/login/", data, login=True):
  File "/usr/local/lib/python3.9/site-packages/instagrapi/mixins/private.py", line 304, in private_request
    self.challenge_resolve(self.last_json)
  File "/usr/local/lib/python3.9/site-packages/instagrapi/mixins/challenge.py", line 68, in challenge_resolve
    return self.challenge_resolve_simple(challenge_url)
  File "/usr/local/lib/python3.9/site-packages/instagrapi/mixins/challenge.py", line 372, in challenge_resolve_simple
    self._send_private_request(challenge_url, {"choice": CHOICE_EMAIL})
  File "/usr/local/lib/python3.9/site-packages/instagrapi/mixins/private.py", line 239, in _send_private_request
    raise ClientNotFoundError(e, response=e.response, **last_json)
instagrapi.exceptions.ClientNotFoundError: 404 Client Error: Not Found for url: https://i.instagram.com/api/challenge/7182357309/K2scazkHeQ/
```

**Comment by adw0rd** [comment]
@quintendewilde I'm rewrite the mechanism for challenge resolving now, most likely this will help you

**Comment by adw0rd** [comment]
Fixed, use `instagrapi==1.11.1`
